### PR TITLE
Create separate google and github sigin buttons

### PR DESCRIPTION
**Pull request by deschantkn**
### Description
Created a google signin button

### Screenshots
Screen Shot 2019-10-08 at 11 48 58

## New Behavior
N/A

### CODE REVIEW

**Comment by byliuyang** [comment]
Hi @deschantkn, thank you very much for working on this feature! Please address automated code review comment from Codacy.

File: app/adapter/web/src/component/pages/shared/SignIn.tsx [hunk 1]
```diff
@@ -16,16 +17,9 @@ export class SignIn extends Component<Props> {
...
-            <img
-              alt={'Sign in with github account'}
-              className={'icon'}
-              src={githubLogo}
-            />{' '}
-            Sign In with Github
-          </div>
-        </a>
+        <GithubSigninButton signInLink={this.props.githubSignInLink} />
+        or
```
Comment (byliuyang) [inline app/adapter/web/src/component/pages/shared/SignIn.tsx]:
```suggestion
```

File: app/adapter/web/src/component/pages/shared/GithubSigninButton.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      <a href={this.props.signInLink}>
+        <div className={'button github'}>
+          <img
+            alt={'Sign in with github account'}
+            className={'icon'}
+            src={githubLogo}
+          />{' '}
+          Sign In with Github
+        </div>
+      </a>
```
Comment (byliuyang) [inline app/adapter/web/src/component/pages/shared/GithubSigninButton.tsx]: Please extract common code used between `GithubSigninButton` and `GoogleSigninButton` and  into `SignInButton` component to encourage code reuse.

File: app/adapter/web/src/component/pages/shared/GithubSigninButton.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+
+import githubLogo from './github.svg';
+
+interface Props {
+  signInLink: string;
+}
+
+export class GithubSigninButton extends Component<Props> {
```
Comment (byliuyang) [inline app/adapter/web/src/component/pages/shared/GithubSigninButton.tsx]:
```suggestion
export class GithubSignInButton extends Component<Props> {
```

File: app/adapter/web/src/component/pages/shared/GithubSigninButton.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  render() {
+    return (
+      <a href={this.props.signInLink}>
+        <div className={'button github'}>
+          <img
+            alt={'Sign in with github account'}
+            className={'icon'}
+            src={githubLogo}
+          />{' '}
```
Comment (byliuyang) [inline app/adapter/web/src/component/pages/shared/GithubSigninButton.tsx]:
```suggestion
          />&nbsp;
```

File: app/adapter/web/src/component/pages/shared/GoogleSigninButton.tsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+export class GoogleSigninButton extends Component<Props> {
+  render() {
+    return (
+      <a href={this.props.signInLink}>
+        <div className={'button google'}>
+          <img
+            alt={'Sign in with google account'}
+            className={'icon'}
+            src={googleLogo}
+          />{' '}
```
Comment (byliuyang) [inline app/adapter/web/src/component/pages/shared/GoogleSigninButton.tsx]:
```suggestion
          />&nbsp;
```

File: app/adapter/web/src/component/pages/shared/google.svg [hunk 6]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 18.1.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (byliuyang) [inline app/adapter/web/src/component/pages/shared/google.svg]:
```suggestion
```

**Comment by deschantkn** [comment]
All good? @byliuyang

File: app/adapter/web/src/component/pages/shared/SignIn.tsx [hunk 7]
```diff
@@ -16,16 +17,9 @@ export class SignIn extends Component<Props> {
...
-            <img
-              alt={'Sign in with github account'}
-              className={'icon'}
-              src={githubLogo}
-            />{' '}
-            Sign In with Github
-          </div>
-        </a>
+        <GithubSignInButton signInLink={this.props.githubSignInLink} />
+        or
```
Comment (dedifferentiator) [inline app/adapter/web/src/component/pages/shared/SignIn.tsx]: Maybe I'm too concerned about such things, but I think word "or" is a bit superfluous here

Comment (byliuyang) [reply app/adapter/web/src/component/pages/shared/SignIn.tsx]: Agreed.

File: app/adapter/web/src/component/pages/shared/GithubSigninButton.tsx (hunk 2 above)
Comment (byliuyang) [reply app/adapter/web/src/component/pages/shared/GithubSigninButton.tsx]: I mean composition

**Comment by byliuyang** [comment]
Thanks for working on this cool feature. Please address all the comments.

**Comment by byliuyang** [comment]
Hi @deschantkn , a large refactor of the code base happened recently. I'll help with this PR.

**Pull request closed without merging**